python/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name='MDSplus',
+    version='7.139.40',
```

Comment:
We'll need to pull the version either from a file or from the git tags

Reply:
I am open to suggestions.  Preferably this can be something dynamic, but filled in when a release is made.

Reply:
Perhaps [`setuptools_scm`](https://pypi.org/project/setuptools-scm/)?